### ISSUE TITLE
Long polling fails in IPAD version 6.0.1

### ISSUE THREAD

**Issue by javayas** [Atmosphere/atmosphere#1021]
I am using atmosphere.js version 0.7.The fallback Transport is streaming
and transport is long-polling.I have a Iframe with previous and next navigations. Whenever I navigate the atmosphere response is published for all the desktop clients.But this request is  on hold in the Ipad 6.0.1(safari 6.0),and the response is sent to the Ipad client only after the time out. Can anyone please guide me on this.Or is there a way to unsubscribe and subscribe back the request.

Version 0.7 has only close method.Is there any other way to overcome this issue in IPAD 6.01 safari browser alone.This functionality is working fine in IPAD Chrome browser.Can anyone help me please.I am new to this technology.Thanks in advance...!!!!
